src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -502,7 +509,11 @@ public function deleteProductAction(Request $request, int $cartId): JsonResponse
      */
     private function getCartInfo(int $cartId): CartInformation
     {
-        return $this->getQueryBus()->handle(new GetCartInformation($cartId));
+        return $this->getQueryBus()->handle(
+            (new GetCartForOrderCreation($cartId))
+                ->setSeparateGiftProducts(false)
+                ->setSeparateGiftCartRules(false)
```

Comment:
Don't you mean to use `true` here? Otherwise you don't need to call the setters with default values.